containers/jersey2-routing/src/main/java/com/yahoo/gondola/container/ClusterIdCallback.java

```diff
@@ -26,10 +26,10 @@
  */
 public interface ClusterIdCallback {
     /**
-     * The callback method to get gondola cluster ID based on request
+     * Returns the Gondola cluster ID based on request.
      *
      * @param request
-     * @return Gondola Cluster ID
+     * @return non-null Gondola cluster ID
```

Comment:
One use case is that if this is a POST request and doesn't have any ID, we need a mechanism to tell the request is this type or not. One implementation could be return a `null` clusterId